tests/python/run_python_tests.sh

```diff
@@ -28,6 +28,7 @@ rm log.log screen.log
 rm -r test_db
 
 ../../vdms -cfg config-tests.json > screen.log 2> log.log &
+sleep 1
```

Comment:
We should investigate a reliable way to do this syncronization.

Reply:
good point. Done.